UnitTests/GitUITests/UserControls/BlameControlTests.cs

```diff
@@ -51,8 +51,9 @@ public void SetUp()
 
             _gitBlameLine = new GitBlameLine(blameCommit1, 1, 1, "line1");
 
-            _sut = new BlameControl().GetTestAccessor();
-            _sut.Blame = new GitBlame(new GitBlameLine[]
+            _blameControl = new BlameControl();
+            var testAccessor = _blameControl.GetTestAccessor();
```

Comment:
💡 Would be better to inline this variable to the next line

Reply:
This is C# 8 syntax and does not compile on AppVeyor with MSVS2017.

Reply:
😕 Even older versions of C# should be able to replace `testAccessor` with `_blameControl.GetTestAccessor()`

Reply:
Come on, I've already wrote you the regarding error message.